### PR TITLE
OB-10 Downloaded feed can be sent to RabbitMQ queue

### PR DESCRIPTION
**Pull request by johnjs**
Hi,
that's the next super-exciting feature! :balloon: This time I added a possibility to redirect the feed fetched from twitter/facebook API to RabbitMQ :rabbit:  The mechanism has been implemented using `streams` and `pipes` :toilet:
Having the queue as a default feed output is going to be the main solution used on the production server. Nevertheless the option to redirect data to JSON file will be still supported as it's quite handy when the scrapers are tested.

To sum up, choose the feed destination and set it in your `.env` file:
`FEED_DESTINATION = [FILE|QUEUE]`

The service consuming the data sent to the queue will be implemented in a separate repository. For the time being, you can test the flow using a worker script implemented [here](185fb9f04). Just run in as a separate process :movie_camera:.

[--> Trello ticket](https://trello.com/c/rOT7G4kI/10-ob-10-scrapers-should-send-fetching-results-to-the-queue)

### CODE REVIEW

**Comment by johnjs** [comment]
:ship:
